specifyweb/frontend/js_src/lib/components/Attachments/index.tsx

```diff
@@ -237,16 +239,23 @@ function Attachments({
         <span className="-ml-2 flex-1" />
         {/* Don't display scale if in dialog to not have resizing/glitching issue */}
         {isInDialog === undefined && (
-          <Label.Inline>
-            {attachmentsText.scale()}
-            <Input.Generic
-              max={maxScale}
-              min={minScale}
-              type="range"
-              value={scale}
-              onValueChange={(value) => setScale(Number.parseInt(value))}
-            />
-          </Label.Inline>
+          <>
+            <Label.Inline>
+              {attachmentsText.scale()}
+              <Input.Generic
+                max={maxScale}
+                min={minScale}
+                type="range"
+                value={scale}
+                onValueChange={(value) => setScale(Number.parseInt(value))}
+              />
+            </Label.Inline>
+            <Button.Small
```

Comment:
Consider making this button much more prominent. Maybe green or blue?
Bulk attachment import is a big new feature after all, so consider make it more visible?

Reply:
and consider renaming "Import" to "Bulk Import" or something else
my problem is that seeing the "Import" button ("import" being a verb) makes me thing that it will let me initialize import right now.
but the button actually opens a list of data sets, including allowing me to resume a previous import
so positioning this as a tool (like workbench), rather than action ("import") seems more appropriate.
does it make sense?

Reply:
**Bulk Import** <-- New name